<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="NetworkView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_to_database"/>
    <addaction name="actionLoad_from_database"/>
    <addaction name="actionClear"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionAllign_to_grid"/>
    <addaction name="separator"/>
    <addaction name="actionZoom"/>
    <addaction name="actionZoom_2"/>
    <addaction name="actionFit_to_screen"/>
    <addaction name="actionZoom_100"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="text">
    <string>Zoom +</string>
   </property>
  </action>
  <action name="actionZoom_2">
   <property name="text">
    <string>Zoom -</string>
   </property>
  </action>
  <action name="actionFit_to_screen">
   <property name="text">
    <string>Fit to screen</string>
   </property>
  </action>
  <action name="actionZoom_100">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Scale 100%</string>
   </property>
  </action>
  <action name="actionSave_to_database">
   <property name="text">
    <string>Save to database...</string>
   </property>
  </action>
  <action name="actionLoad_from_database">
   <property name="text">
    <string>Load from database...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About visualization</string>
   </property>
  </action>
  <action name="actionAllign_to_grid">
   <property name="text">
    <string>Allign to grid</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Close network</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NetworkView</class>
   <extends>QGraphicsView</extends>
   <header>networkview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
